Xamarin.Forms.Platform.Android/FastRenderers/FrameRenderer.cs

```diff
@@ -236,13 +236,16 @@ protected virtual void OnElementPropertyChanged(object sender, PropertyChangedEv
 				UpdateClippedToBounds();
 		}
 
-		void UpdateClippedToBounds() => this.SetClipToOutline(Element.IsClippedToBounds);
+		void UpdateClippedToBounds()
```

Comment:
It's not implemented for the regular renderer?

Reply:
Hrs, regular renderer uses VisualTracker do apply this property.